src/librustc_span/lib.rs

```diff
@@ -8,9 +8,6 @@
 #![feature(crate_visibility_modifier)]
 #![feature(nll)]
 #![feature(optin_builtin_traits)]
-#![feature(rustc_attrs)]
-#![feature(specialization)]
```

Comment:
This appears to have caused a failure in rollup https://github.com/rust-lang/rust/pull/68971: my PR https://github.com/rust-lang/rust/pull/68718 requires specialization to be enabled.

Reply:
Added it back in the latest commit.